packages/mobile/src/exchange/ExchangeConfirmationCard.test.tsx

```diff
@@ -3,25 +3,33 @@ import * as React from 'react'
 import 'react-native'
 import { Provider } from 'react-redux'
 import ExchangeConfirmationCard from 'src/exchange/ExchangeConfirmationCard'
-import { createMockStore } from 'test/utils'
+import { createMockStore, getElementText } from 'test/utils'
 
 const localAmount = {
   value: '1.23',
   exchangeRate: '0.555',
   currencyCode: 'EUR',
 }
-const makerAmount = { value: '20', currencyCode: 'cGLD', localAmount }
-const takerAmount = { value: '1.99', currencyCode: 'cUSD', localAmount }
 
 const store = createMockStore({})
 
-it('renders correctly with no exchange rate', () => {
+it('renders correctly', () => {
   const tree = render(
     <Provider store={store}>
-      <ExchangeConfirmationCard makerAmount={makerAmount} takerAmount={takerAmount} />
+      <ExchangeConfirmationCard
+        makerAmount={{ value: '20', currencyCode: 'cGLD', localAmount }}
+        takerAmount={{ value: '1.99', currencyCode: 'cUSD', localAmount }}
+      />
     </Provider>
   )
   expect(tree).toMatchSnapshot()
+
+  expect(getElementText(tree.getByTestId('CeloAmount/value'))).toEqual('20.000')
+  expect(getElementText(tree.getByTestId('CeloExchangeRate/value'))).toEqual('€0.55')
+
+  expect(getElementText(tree.getByTestId('TotalLineItem/Total/value'))).toEqual('$2.64')
```

Comment:
nit: I'm not sure how these values are obtained, maybe a small comment would help to understand it

Reply:
yep you're right it's quite confusing given the fake localAmount used for testing.